core/iwasm/include/wasm_export.h

```diff
@@ -800,6 +800,27 @@ wasm_runtime_call_wasm_v(wasm_exec_env_t exec_env,
                          uint32_t num_results, wasm_val_t results[],
                          uint32_t num_args, ...);
 
+/**
+ * Call a function reference of a given WASM runtime instance with
+ * arguments.
+ *
+ * @param exec_env the execution environment to call the function
+ *   which must be created from wasm_create_exec_env()
+ * @param element_indices the function ference indicies, usually
```

Comment:
Not sure whether element_indice is better?
And `the function ference indicies` to `the function reference indice`?

Reply:
i just moved this comment from another file. anyway, fixed.

Reply:
yes, it was committed by the some community developer before, we should review the code more strictly and let him change :)